app/style/MyTickets.less

```diff
@@ -32,7 +32,7 @@
 }
 
 .ticket-card.is-expanded {
-  height: 100%;
+  height: 165px;
```

Comment:
Returning this to 165px is breaking the height on higher-width displays (eg: when the wallet's window width is 1240px).